stencil-library/src/components/refund-form/refund-form.scss

```diff
@@ -0,0 +1,99 @@
+@import 'bootstrap-grid';
+@import 'bootstrap-utilities';
+
+:host {
+  @include host-base-styles;
+  display: block;
+}
+
+.info-text-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1rem;
+  border-top: var(--jfi-form-control-border-color) solid 1px;
+  border-bottom: var(--jfi-form-control-border-color) solid 1px;
+  margin-bottom: 1rem;
+}
+
+.info-icon {
+  margin-left: 1rem;
+}
+
+.info-text {
+  flex: 1;
+  padding: 1rem;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  color: var(--jfi-form-control-color);
+}
+
+*::part(input) {
+  font-family: var(--jfi-layout-font-family);
+  background-color: var(--jfi-form-control-background-color);
+  border-color: var(--jfi-form-control-border-color);
+  border-width: var(--jfi-form-control-border-width);
+  border-bottom-width: var(--jfi-form-control-border-bottom-width);
+  border-left-width: var(--jfi-form-control-border-left-width);
+  border-right-width: var(--jfi-form-control-border-right-width);
+  border-top-width: var(--jfi-form-control-border-top-width);
+  border-radius: var(--jfi-form-control-border-radius);
+  border-style: var(--jfi-form-control-border-style);
+  box-shadow: var(--jfi-form-control-box-shadow);
+  color: var(--jfi-form-control-color);
+  font-size: var(--jfi-form-control-font-size);
+  font-weight: var(--jfi-form-control-font-weight);
+  line-height: var(--jfi-form-control-line-height);
+  margin: var(--jfi-form-control-margin);
+  padding: var(--jfi-form-control-padding);
+
+  &:focus {
+    color: var(--jfi-form-control-color-focus);
+    border-color: var(--jfi-form-control-border-color-focus);
+    box-shadow: var(--jfi-form-control-box-shadow-focus);
+  }
+}
+
+.btn.jfi-submit-button {
```

Comment:
Are we able just to use bootstrap `btn`? I know that in the `payment-form` we do something similar to this but I am not sure why it was done that way. We should use bootstrap directly and export anything we want customers to be able to override

Reply:
You mean, the button would still be styled with the jfi css variables, but the selector would be just `.btn`?

Reply:
I mean that we should use the `.btn` bootstrap class to apply styles for the button, and if a customer wants to customize it we can use the css exports. I think the payment form should have done that as well, and we would need less variables. Maybe there is a reason it was done this way though